File: test/EntityFramework/VBTests/_VbMonsterModel/ExternalTypes/PhoneMm.vb
'---------------------------------------------------------------------
' <copyright file="PhoneMm.vb" company="Microsoft">
'      Copyright (c) Microsoft Corporation.  All rights reserved.
' </copyright>
'
' @owner       avickers
' @backupOwner bricelam
'---------------------------------------------------------------------

Imports System

Namespace Another.Place
    Partial Public Class PhoneMm
        Public Property PhoneNumber As String
        Public Property Extension As String = "None"
        Public Property PhoneType As PhoneTypeMm
    End Class
End Namespace

File: test/EntityFramework/FunctionalTests/_TestModels/TemplateModels/CsMonsterModel/ExternalTypes/PhoneMm.cs
namespace Another.Place
{
    public partial class PhoneMm
    {
        public PhoneMm()
        {
            Extension = "None";
        }

        public string PhoneNumber { get; set; }
        public string Extension { get; set; }
        public PhoneTypeMm PhoneType { get; set; }
    }
}